lib/cfg-lexer.c

```diff
@@ -818,10 +818,9 @@ cfg_lexer_lex(CfgLexer *self, YYSTYPE *yylval, YYLTYPE *yylloc)
           *yylloc = self->include_stack[self->include_depth].lloc;
           tok = token->type;
           if (token->type == LL_TOKEN)
-            {
-              tok = token->token;
-              injected = TRUE;
-            }
+            tok = token->token;
```

Comment:
I would use an if-else statement for setting the `tok` variable. (Resetting after 1 line does not make sense.)